xarray/tests/test_dataarray.py

```diff
@@ -2544,4 +2530,3 @@ def test_rolling_reduce(da, center, min_periods, window, name):
     actual = rolling_obj.reduce(getattr(np, 'nan%s' % name))
     expected = getattr(rolling_obj, name)()
     assert_allclose(actual, expected)
-    assert da.dims == expected.dims
```

Comment:
I thought `assert_allclose(actual, expected)` does the same thing, so I deleted it.
I experientially found that this function raises an error if two `dims`'s orders are differ.
Wasn't `assert_allclose` designed to do it?